### ISSUE TITLE
A likely linking problem

### ISSUE THREAD

**Issue by jimyu94** [ejgallego/coq-serapi#42]
I am using Coq 8.7.1, OCaml 4.06.0. After compiling CompCert 3.2, if I run
```
sertop -R lib,compcert.lib -R common,compcert.common -R x86_64,compcert.x86_64 -R x86,compcert.x86 -R backend,compcert.backend -R cfrontend,compcert.cfrontend -R driver,compcert.driver -R flocq,compcert.flocq -R exportclight,compcert.exportclight -R cparser,compcert.cparser
```
and then `(Add () "From compcert Require Import Ordered.")`, an error appears:
```
(Feedback((id 2)(route 0)(contents(Message Error(((fname"")(line_nb 17)(bol_pos 999)(line_nb_last 17)(bol_pos_last 999)(bp 999)(ep 1058)))(Pp_box(Pp_hovbox 0)(Pp_glue((Pp_string while)(Pp_print_break 1 0)(Pp_string loading)(Pp_print_break 1 0)(Pp_string /Users/wolf/.opam/4.06.0/lib/coq/plugins/micromega/micromega_plugin.cmxs:)(Pp_print_break 1 0)(Pp_string"error loading shared library: dlopen(/Users/wolf/.opam/4.06.0/lib/coq/plugins/micromega/micromega_plugin.cmxs, 10): Symbol not found: _camlBig_int\n  Referenced from: /Users/wolf/.opam/4.06.0/lib/coq/plugins/micromega/micromega_plugin.cmxs\n  Expected in: flat namespace\n in /Users/wolf/.opam/4.06.0/lib/coq/plugins/micromega/micromega_plugin.cmxs"))))))))
```
This looks like a linking problem in SerAPI, as `coqtop` doesn't have the problem.
If I run
```
coqtop -R lib compcert.lib -R common compcert.common -R x86_64 compcert.x86_64 -R x86 compcert.x86 -R backend compcert.backend -R cfrontend compcert.cfrontend -R driver compcert.driver -R flocq compcert.flocq -R exportclight compcert.exportclight -R cparser compcert.cparser
```
Then `From compcert Require Import Ordered.`, I got 
```
[Loading ML file z_syntax_plugin.cmxs ... done]
[Loading ML file quote_plugin.cmxs ... done]
[Loading ML file newring_plugin.cmxs ... done]
[Loading ML file omega_plugin.cmxs ... done]
[Loading ML file extraction_plugin.cmxs ... done]
[Loading ML file recdef_plugin.cmxs ... done]
[Loading ML file romega_plugin.cmxs ... done]
[Loading ML file r_syntax_plugin.cmxs ... done]
[Loading ML file fourier_plugin.cmxs ... done]
[Loading ML file micromega_plugin.cmxs ... done]
```
which looks fine.

**Comment by ejgallego** [comment]
Should be fixed by 7a38201b290f97f46ffa83fcea82a5ec1c70ddff, let me know how it goes.

**Comment by jimyu94** [comment]
I confirm this bug has been solved. Thanks!
